### PR TITLE
Update Microsoft Azure Storage Connector Example 2.x page

### PR DESCRIPTION
**Pull request by KalinduGandara**

### CODE REVIEW

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 1]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
 
 > **Note**: Before invoking the API, you need to create a **Storage Account** in **Microsoft Azure Storage account**. See [Azure Storage Configuration]({{base_path}}/reference/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration/) documentation for more information.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration Project
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:20]:
```suggestion
## Set up the Integration project
```

Comment (anupama-pathirage) [reply en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:20]: Also use sentence case for titles. Check and fix all places. https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 2]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
-
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!}
+Follow the steps in the [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.
 
 ## Creating the Integration Logic
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:24]:
```suggestion
## Create the Integration Logic
```

Comment (anupama-pathirage) [reply en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:24]: Use sentence case.

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 3]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-1. Specify the API name as `MSAzureStorageTestAPI` and API context as `/azure`.
+1. Click `+` on the Extension panel APIs to create the REST API. Specify the API name as `MSAzureStorageTestAPI` and the API context as `/azure`. Then delete the default `/resource` endpoint.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:26]:
```suggestion
1. Click `+` on the Extension panel APIs to create the REST API. Specify the API name as `MSAzureStorageTestAPI` and the API context as `/azure`. Then, delete the default `/resource` endpoint.
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 4]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
+    - **Connection Name**: Unique name to identify the connection.
+    - **Connection Type**: msazurestorage
+    - **Account Name**: The name of the Azure storage account.
+    - **Client ID**: The client ID of the application.
+    - **Client Secret**: The client Secret of the application.
+    - **Tenant ID**: The Tenant ID of the application.
+    - **Default Endpoints Protocol**: http
     !!! note
         You can either define the Account Access key or Client Credentials for authentication. For more information, please refer [Initialize the connector guide]({{base_path}}/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-reference/#initialize-the-connector).
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:52]:
```suggestion
        You can either define the Account Access key or Client Credentials for authentication. For more information, please refer to [Initialize the connector guide]({{base_path}}/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-reference/#initialize-the-connector).
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 5]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-7. Drag and drop the [Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/) to send back the response from creating the container as shown below.
+7. Click `+` below the `CreateContainer` node to add the [Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/) to send back the response from creating the container as shown below.
+
 
     <img src="{{base_path}}/assets/img/integrate/connectors/msazure-connector-2x/adding_respond_mediator.png" title="Adding a respond mediator" width="800" alt="Microsoft Azure Storage use case"/>
 
-8. Create the next API resource, which is `/addblob` by dragging and dropping another API resource to the design view. This API resource will retrieve information about the blob from the incoming HTTP POST request such as the container name, blob name and the file content and upload it to Microsoft Azure Storage.
+8. Create the next API resource, which is `/addblob`, by following the same steps as before. This API resource will retrieve information about the blob from the incoming HTTP POST request, such as the container name, blob name, and the file content, and upload it to Microsoft Azure Storage.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:70]:
```suggestion
8. Follow the same steps to create the next API resource, '/add blob'. This API resource will retrieve information about the blob from the incoming HTTP POST request, such as the container name, blob name, and the file content, and upload it to Microsoft Azure Storage.
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 6]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
     - Text Content - json-eval($.textContent)
     - Metadata - json-eval($.metadata)
 
-10. Drag and drop the [Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/) to send back the response from uploading the blob.
+10. Add the [Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/) to send back the response from uploading the blob.
 
     <img src="{{base_path}}/assets/img/integrate/connectors/msazure-connector-2x/configure_add_blob_operation.png" title="Configuring upload blob operation" width="800" alt="Microsoft Azure Storage use case"/>
 
-11. Create the next API resource, which is `/downloadblob` by dragging and dropping another API resource to the design view. This API resource will retrieve information from the incoming HTTP POST request such as the container name and blob name and download from Microsoft Azure Storage.
+11. Create the next API resource, which is `/downloadblob`, by following the same steps as before. This API resource will retrieve information from the incoming HTTP POST request, such as the container name and blob name, and download from Microsoft Azure Storage.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:83]:
```suggestion
11. Follow the same steps to create the next API resource, '/download blob'. This API resource will retrieve information from the incoming HTTP POST request, such as the container name and blob name, and download from Microsoft Azure Storage.
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md (hunk 5 above)
Comment (anupama-pathirage) [reply en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:70]: Fix all the similar cases.

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 7]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
 
 > **Note**: Before invoking the API, you need to create a **Storage Account** in **Microsoft Azure Storage account**. See [Azure Storage Configuration]({{base_path}}/reference/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration/) documentation for more information.
 
-## Configure the connector in WSO2 Integration Studio
+## Set up the Integration project
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:20]:
```suggestion
## Set up the integration project
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 8]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
+Follow the steps in the [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.
 
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!}
+## Create the Integration logic
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:24]:
```suggestion
## Create the integration logic
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 9]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-8. Create the next API resource, which is `/addblob` by dragging and dropping another API resource to the design view. This API resource will retrieve information about the blob from the incoming HTTP POST request such as the container name, blob name and the file content and upload it to Microsoft Azure Storage.
+8. Follow the same steps to create the next API resource, `/addblob`. This API resource will retrieve information about the blob from the incoming HTTP POST request, such as the container name, blob name, and the file content, and upload it to Microsoft Azure Storage.
 
-9. Drag and drop the ‘uploadBlob’ operation of the Microsoft Azure Storage Connector to the Design View. In the properties view, select the already created connection as 'Connection' from the drop down menu and provide the following expressions to the below properties,
+9. Next, add the `uploadBlob` operation from the **Externals** tab using the newly created connection. In the properties view, provide the following expressions for the below properties:
     - Container Name - json-eval($.containerName)
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:73]: Use backtick for property names

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 10]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-8. Create the next API resource, which is `/addblob` by dragging and dropping another API resource to the design view. This API resource will retrieve information about the blob from the incoming HTTP POST request such as the container name, blob name and the file content and upload it to Microsoft Azure Storage.
+8. Follow the same steps to create the next API resource, `/addblob`. This API resource will retrieve information about the blob from the incoming HTTP POST request, such as the container name, blob name, and the file content, and upload it to Microsoft Azure Storage.
 
-9. Drag and drop the ‘uploadBlob’ operation of the Microsoft Azure Storage Connector to the Design View. In the properties view, select the already created connection as 'Connection' from the drop down menu and provide the following expressions to the below properties,
+9. Next, add the `uploadBlob` operation from the **Externals** tab using the newly created connection. In the properties view, provide the following expressions for the below properties:
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:72]:
```suggestion
9. Next, add the `uploadBlob` operation from the **Externals** tab using the created connection. In the properties view, provide the following expressions for the below properties:
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 11]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-11. Create the next API resource, which is `/downloadblob` by dragging and dropping another API resource to the design view. This API resource will retrieve information from the incoming HTTP POST request such as the container name and blob name and download from Microsoft Azure Storage.
+11. Follow the same steps to create the next API resource, `/downloadblob`. This API resource will retrieve information from the incoming HTTP POST request, such as the container name and blob name, and download from Microsoft Azure Storage.
 
-12. Next drag and drop the ‘downloadBlob’ operation of the Microsoft Azure Storage Connector to the Design View. In the properties view, select the already created connection as 'Connection' from the drop down menu and provide the following expressions to the below properties,
+12. Next, add the `downloadBlob` operation from the **Externals** tab using the newly created connection. In the properties view, provide the following expressions for the below properties:
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:85]:
```suggestion
12. Next, add the `downloadBlob` operation from the **Externals** tab using the created connection. In the properties view, provide the following expressions for the below properties:
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 12]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
     - Container Name - json-eval($.containerName)
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:87]: Use backtick for property names

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 13]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-14. Create the next API resource, which is `/deleteblob` by dragging and dropping another API resource to the design view. This API resource will retrieve information from the incoming HTTP POST request such as the container name and blob name and delete the blob from Microsoft Azure Storage.
+14. Follow the same steps to create the next API resource, `/deleteblob`. This API resource will retrieve information from the incoming HTTP POST request, such as the container name and blob name, and delete the blob from Microsoft Azure Storage.
 
-15. Next drag and drop the ‘deleteBlob’ operation of the Microsoft Azure Storage Connector to the Design View. In the properties view, select the already created connection as 'Connection' from the drop down menu and provide the following expressions to the below properties,
+15. Next, add the `deleteBlob` operation from the **Externals** tab using the newly created connection. In the properties view, provide the following expressions for the below properties:
 
     - Container Name - json-eval($.containerName)
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:98]: Use backtick for property names

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 14]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-14. Create the next API resource, which is `/deleteblob` by dragging and dropping another API resource to the design view. This API resource will retrieve information from the incoming HTTP POST request such as the container name and blob name and delete the blob from Microsoft Azure Storage.
+14. Follow the same steps to create the next API resource, `/deleteblob`. This API resource will retrieve information from the incoming HTTP POST request, such as the container name and blob name, and delete the blob from Microsoft Azure Storage.
 
-15. Next drag and drop the ‘deleteBlob’ operation of the Microsoft Azure Storage Connector to the Design View. In the properties view, select the already created connection as 'Connection' from the drop down menu and provide the following expressions to the below properties,
+15. Next, add the `deleteBlob` operation from the **Externals** tab using the newly created connection. In the properties view, provide the following expressions for the below properties:
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:96]:
```suggestion
15. Next, add the `deleteBlob` operation from the **Externals** tab using the created connection. In the properties view, provide the following expressions for the below properties:
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 15]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-17. Create the next API resource, which is `/listmetadata` by dragging and dropping another API resource to the design view. This API resource will retrieve information from the incoming HTTP POST request such as the container name and blob name and retrieve the metadata of the blob from Microsoft Azure Storage.
+17. Follow the same steps to create the next API resource, `/listmetadata`. This API resource will retrieve information from the incoming HTTP POST request, such as the container name and blob name, and retrieve the metadata of the blob from Microsoft Azure Storage.
 
-18. Next drag and drop the ‘listMetadata’ operation of the Microsoft Azure Storage Connector to the Design View. In the properties view, select the already created connection as 'Connection' from the drop down menu and provide the following expressions to the below properties,
+18. Next, add the `listMetadata` operation from the **Externals** tab using the newly created connection. In the properties view, provide the following expressions for the below properties:
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:107]:
```suggestion
18. Next, add the `listMetadata` operation from the **Externals** tab using the created connection. In the properties view, provide the following expressions for the below properties:
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 16]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
     - Container Name - json-eval($.containerName)
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:109]: Use backtick for property names

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 17]
```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
...
 
-20. Create the next API resource, which is `/deletecontainer` by dragging and dropping another API resource to the design view. This API resource will retrieve information from the incoming HTTP POST request such as the container name and delete the container from Microsoft Azure Storage.
+20. Follow the same steps to create the next API resource, `/deletecontainer`. This API resource will retrieve information from the incoming HTTP POST request, such as the container name, and delete the container from Microsoft Azure Storage.
 
-21. Next drag and drop the ‘deleteContainer’ operation of the Microsoft Azure Storage Connector to the Design View. In the properties view, select the already created connection as 'Connection' from the drop down menu and provide the following expressions to the below properties,
+21. Next, add the `deleteContainer` operation from the **Externals** tab using the newly created connection. In the properties view, provide the following expressions for the below properties:
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:118]:
```suggestion
21. Next, add the `deleteContainer` operation from the **Externals** tab using the created connection. In the properties view, provide the following expressions for the below properties:
```

File: en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md [hunk 18]
```diff
@@ -216,17 +208,7 @@ You can download the ZIP file and extract the contents to get the project code.
...
-
-??? note "Click here for instructions on deploying on WSO2 Enterprise Integrator 6"
-    1. You can copy the composite application to the `<PRODUCT-HOME>/repository/deployment/server/carbonapps` folder and start the server.
-
-    2. WSO2 EI server starts and you can login to the Management Console https://localhost:9443/carbon/ URL. Provide login credentials. The default credentials will be admin/admin. 
-
-    3. You can see that the API is deployed under the API section. 
+To deploy and run the project, refer to the [Build and Run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.
 
 ## Testing
```
Comment (chanikag) [inline en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md:213]:
```suggestion
## Test
```

**Review by chanikag — APPROVED** [review]

**Pull request merged**